ftw/publisher/receiver/tests/test_getData.py

```diff
@@ -0,0 +1,39 @@
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.publisher.receiver.tests import IntegrationTestCase
+from ftw.publisher.receiver.browser.views import GetSavedData
+from plone.app.testing import TEST_USER_NAME
+from plone.app.testing import login
+from Products.CMFCore.utils import getToolByName
+import transaction
+
+
+class TestGetData(IntegrationTestCase):
+
+    def setUp(self):
+        super(TestGetData, self).setUp()
+
+        self.grant('Manager')
+        self.types_tool = getToolByName(self.portal, 'portal_types')
+        self.types_tool['Plone Site'].allowed_content_types = ('FormFolder',)
+        transaction.commit()
```

Comment:
I would probably move this to the end of `setUp`; I believe that the `setSavedFormInput` call is not commited, is it?